test/run-k8s-integration.sh

```diff
@@ -44,4 +44,10 @@ else
   base_cmd="${base_cmd} --gce-region=${gce_region}"
 fi
 
+# TODO: When beta csi snapshotter sidecar is promoted to stable, add stable
```

Comment:
I think we can eventually enable it on all overlays so we don't need to have an overlay check.

Reply:
If we remove this check right now --snapshotclass-file will be passed as input to main.go, and stable driver running against >=1.17 will be running snapshot tests and would fail. hence I put a todo.

Reply:
yes, I was referring to the todo comment that says we should add the other overlays to the check. I'm thinking we can do the opposite and remove the check once we support snapshots in all the overlays.

Reply:
Ah got it. Yes if we plan to put it in all, we can remove the check completely eventually.